nemo/utils/exp_manager.py

```diff
@@ -555,3 +561,12 @@ def on_train_end(self, trainer, pl_module):
     trainer.configure_checkpoint_callback(checkpoint_callback)
     trainer.callbacks.append(checkpoint_callback)
     trainer.checkpoint_callback = checkpoint_callback
+
+
+def add_callbacks(trainer: 'pytorch_lightning.Trainer', nemo_callbacks: Optional[List[str]]):
+
+    for callback in nemo_callbacks:
+        if callback in callbacks.AVAILABLE_CALLBACKS:
+            trainer.callbacks.append(callbacks.AVAILABLE_CALLBACKS[callback])
+        else:
+            raise NameError(" Request callback is not part of nemo callbacks please check callback name")
```

Comment:
You really should look into hydra.instantiate. The solution works now, because these callbacks take no constructor arguments. But the moment you want to enhance your callback by saying moving `print_freq` to your callback constructor as opposed to taking it from trainer, this method will need to be improved.

Reply:
Agreed, Omry is working on a improved way of hydra instantiate, so will comeback on this when it was finalized 
viz: https://github.com/facebookresearch/hydra/issues/566

Reply:
Moved out of this PR